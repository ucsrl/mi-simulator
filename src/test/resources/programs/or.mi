	SEG
	JUMP s

s:  OR W I 1, I 1, R0
end:	HALT