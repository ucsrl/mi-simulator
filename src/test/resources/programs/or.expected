INS: JUMP 1 + !R15
R15: 0 -> 3
C: 0; V: 0; Z: 0; N: 0
0: 241; 1: 175; 2: 1; 3: 178; 4: 1; 5: 1; 6: 80

INS: OR W I H'00000001', I H'00000001', R0
R0: 0 -> 1; R15: 3 -> 7
C: 0; V: 0; Z: 0; N: 0
0: 241; 1: 175; 2: 1; 3: 178; 4: 1; 5: 1; 6: 80

INS: HALT
R0: 1; R15: 7
C: 0; V: 0; Z: 0; N: 0
0: 241; 1: 175; 2: 1; 3: 178; 4: 1; 5: 1; 6: 80

